lua/autorun/owoifier.lua

```diff
@@ -0,0 +1,67 @@
+local OwOemotes = {"OwO", "owo", "UwU", "uwu", "TwT", ">w<", ":3c", ":3", ":33", ":333", "nya", "(> owo )>",
+					"*pounces on you*", "*glomps u*", "mrowww", "~", ":owo:", ":uwu:", ":awoo:", ":blush:", 
+					":catgirl:", ":pat:", ":snug:", ":thinkerfelix:", ":weebblush1:", ":weebears:", ":widetrap:"} //Add more emotes here
+
+local function OwOifier(text)
+	local text = string.Replace(string.Replace(text,"R","W"),"r","w")
+	local text = string.Replace(string.Replace(text,"L","W"),"l","w")
+
+	for emotecolon in string.gmatch(text, ":%a-:") do //replace swampchat image emotes with text ones, because I cant figure out how to prevent them from being owo-ified
```

Comment:
do you mean the r/l being replaced? maybe you could iterate the string one letter at a time and keep a boolean state if the current letter is part of an emote name?